test/sharness/t0250-files-api.sh

```diff
@@ -521,7 +521,11 @@ test_files_api() {
 
   # test mv
   test_expect_success "can mv dir $EXTRA" '
-    ipfs files mv /cats/this/is /cats/
+    ipfs files mv /cats/this/is /
```

Comment:
We should test moving to _both_ `/cats/` and `/cats`. Maybe move to `/cats/`, then to `/`, then to `/cats`?

Reply:
OK, I have done it.